### PR TITLE
Create Security Policy

### PR DESCRIPTION
**Pull request by TechStudent11**
GitHub recommends that there is a security policy.

Not sure if this falls into the "If it is because of 'best practices', I won't merge it" or however he worded it.

### CODE REVIEW

**Comment by ryanolee** [comment]
Might be worth having an email to responsibility disclose vulnerabilitys to over creating public issues. Though not sure if @ThioJoe  would want to open /maintain one 🤔

**Comment by TechStudent11** [comment]
@ryanolee yeah maybe having a dedicated email set up would be better. If @ThioJoe says so, I can add a new commit to this PR.

**Comment by KendallDoesCoding** [comment]
Yeah, @ThioJoe . I think yall can link  a email like privacy@thiojoe.com on the security policy, as GITHUB recommends that.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Add a email like privacy@thiojoe.com to report a security issue instead of creating a issue on the GITHUB.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+IF YOU FIND A VULNERABILITY: Please create an issue. If you know how to resolve it, please create a PR.
```
Comment (KendallDoesCoding) [inline SECURITY.md:11]: Add a email like privacy@thiojoe.com to report a security issue instead of creating a issue on the GITHUB.

**Review by KendallDoesCoding — COMMENTED** [review]
Add a email like privacy@thiojoe.com to report a security issue instead of creating a issue on the GITHUB.

**Comment by TechStudent11** [comment]
OKAY, I GET IT! @KendallDoesCoding I will change `SECURITY.md` to reflect these changes.

File: SECURITY.md (hunk 1 above)
Comment (TechStudent11) [reply SECURITY.md]: I changed it.

**Comment by KendallDoesCoding** [comment]
@ThioJoe kindly create the email privacy@thiojoe.com from your end to report security issues.

**Review by KendallDoesCoding — COMMENTED** [review]
email is still pending to be made by @ThioJoe .

**Comment by TechStudent11** [comment]
@KendallDoesCoding yep. Maybe he just doesn't to **or** is still thinking about it.

**Comment by KendallDoesCoding** [comment]
Yeah, I just put in the suggestion a long time back, but this request has
not yet been merged or closed, @ThioJoe, please make a decision on this.
Anyway, github does suggest this feature, so even if you don't add the
email, I suggest you merge this to the project.

**Comment by TechStudent11** [comment]
Yeah

**Comment by KendallDoesCoding** [comment]
> Yeah

@TechStudent11 , I don't think @ThioJoe wants to add this, can you please close this pull request?

**Comment by TechStudent11** [comment]
Yeah sure I guess. @ThioJoe can reopen this if he wants.

**Comment by KendallDoesCoding** [comment]
Thanks

**Pull request closed without merging**